### ISSUE TITLE
demo?

### ISSUE THREAD

**Issue by jgcmarins** [leandrino/dashboard-report#11]
does it makes sense to have a live demo?
it could be hosted by gh pages or now.sh

**Comment by leandrino** [comment]
Yes, I'll create a full demo experience when merging the two repos [graphql](https://github.com/leandrino/dashboard-report-graphql) and [this](https://github.com/leandrino/dashboard-report) in a Lerna monorepo
